### PR TITLE
INTLY-7426 - e2e test for checking operator versions

### PR DESCRIPTION
**Pull request by KevFan**
# Description

Move check for operator versions as part of functional tests

Associcated test case PR:
* https://gitlab.cee.redhat.com/integreatly-qe/integreatly-test-cases/merge_requests/192

Jira:
* https://issues.redhat.com/browse/INTLY-7426

## Type of change

- [x] Test automation

## Checklist
- [x] Verified independently on a cluster by reviewer

### CODE REVIEW

File: test/e2e/integreatly_test.go [hunk 1]
```diff
@@ -169,65 +166,6 @@ func waitForProductDeployment(t *testing.T, f *framework.Framework, ctx *framewo
 	return nil
 }
 
-func execToPod(command string, podname string, namespace string, container string, f *framework.Framework) (string, error) {
```
Comment (jjaferson) [inline test/e2e/integreatly_test.go:172]: just out of curiosity why is this being removed? is this being replaced by this one https://github.com/integr8ly/integreatly-operator/blob/master/test/common/shared_functions.go#L38

Comment (KevFan) [reply test/e2e/integreatly_test.go:172]: Yup, the same function got moved there for the tests in `test/common` package and this function wasn't anymore in `test/e2e/integreatly_test.go` so just thought i'll remove while I was there

**Pull request merged**